app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,46 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_by_comment_and_curiosity_by_user_scope, only: [:update, :destroy]
+  before_action :find_curiosity_card, only: [:edit, :create]
+
+  def create
+    @comment = @curiosity.comments.new(comment_params)
+    if @comment.save
+      redirect_to @curiosity
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def edit
+    @comment = Comment.find(params[:id])
+  end
+
+  def update
+    @comment&.update(comment_params)
```

Comment:
Hrmm why would it be possible for record not to exists? 🤔 
I think I would a different behavior, like a `Records doesn't exists` message

Reply:
my bad, I fixed with the previous review of @ingilniero  hehe, Thanks :D